api/api/controllers/default_controller.py

```diff
@@ -23,15 +24,14 @@ async def default_info(pretty=False):
     Returns various basic information about the API
     """
     info_data = load_spec()
-    timestamp = time.strftime("%Y-%m-%dT%H:%M:%S%z", time.gmtime())
     data = {
         'title': info_data['info']['title'],
         'api_version': info_data['info']['version'],
         'revision': info_data['info']['x-revision'],
         'license_name': info_data['info']['license']['name'],
         'license_url': info_data['info']['license']['url'],
         'hostname': socket.gethostname(),
-        'timestamp': timestamp
+        'timestamp': datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
I suggest using global variables for both standard date format and with decimals.